sonar-db/src/main/java/org/sonar/db/version/v53/UpdateCustomDashboardInLoadedTemplates.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.db.version.v53;
+
+import java.sql.SQLException;
+import org.sonar.db.Database;
+import org.sonar.db.version.BaseDataChange;
+import org.sonar.db.version.MassUpdate;
+import org.sonar.db.version.Select;
+import org.sonar.db.version.SqlStatement;
+
+/**
+ * As the default dashboard 'Dashboard' has been renamed to 'Custom', the corresponding entry in LOADED_TEMPLATES is updated,
+ * in order to not have this dashboard twice.
+ */
+public class UpdateCustomDashboardInLoadedTemplates extends BaseDataChange {
+
+  public UpdateCustomDashboardInLoadedTemplates(Database db) {
+    super(db);
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    MassUpdate update = context.prepareMassUpdate().rowPluralName("loaded templates");
+    update.select("SELECT l.id FROM loaded_templates l WHERE l.kee='Dashboard'");
```

Comment:
Missing `and l.template_type='DASHBOARD'` (see constant `LoadedTemplateDto.DASHBOARD_TYPE`)